### DISCUSSION TITLE
Redirecting to upload page

### DISCUSSION THREAD

**Discussion by chinxcode** [community/community#133630] [GitHub Education] [answered]
### Hi everyone,

![screencapture-education-github-discount-requests-15282384-additional-information-2024-07-26-14_09_35 (Small)](https://github.com/user-attachments/assets/3dfd162c-07b9-4874-bb71-b80ac0403dca)

hello everyone, I was applying for GitHub education, after verifying my school email and continuing to the next step for uploading proof of academic status, when I upload my proof and click on process my application, I am being redirected to the same page again and again without any error, I have tried uploading the proof using other devices too but not working. I have tried cancelling and doing the process many times but not working.

Please Help
Thanks

**Comment by hyunbinseo** [comment]
Facing an identical issue.

At first, it seemed to be a location permission related issue:

> To continue, you must modify your browser settings to allow GitHub to user your location information.

![image](https://github.com/user-attachments/assets/f5ca1994-8374-4a1a-9200-374989ba3f4f)

However, even with the location permission issue resolved, the form continuously redirects to itself.

![image](https://github.com/user-attachments/assets/73c8eca0-e59f-452b-b9b4-d51cb47376c3)

**Reply by Rothj** [reply]
I am facing the same issue (and this forum as well: [https://github.com/orgs/community/discussions/133303](https://github.com/orgs/community/discussions/133303))

**Comment by davevad93** [comment]
Hi @chinxcode ,

This is a problem I have noticed that has no general solution. Try the workaround in discussion #126276, if that doesn't work, try another browser like `Brave`. These are the only solutions I have seen that have worked for some users experiencing the same problem.

**Reply by Rothj** [reply, marked as answer]
Using the webcam also worked for me now!

**Comment by chinxcode** [comment]
Webcam worked guyss :)

**Reply by davevad93** [reply]
I'm glad it worked! 👍
